rswag-specs/lib/rswag/specs/request_factory.rb

```diff
@@ -213,14 +213,22 @@ def add_headers(request, metadata, swagger_doc, parameters, example)
           tuples << ['Content-Type', content_type]
         end
 
+        # Host header
+        host = metadata[:operation][:host] || swagger_doc[:host]
+        if host.present?
+          host = example.respond_to?(:'Host') ? example.send(:'Host') : host
```

Comment:
@jtannas I was wondering if you might have meant: `example.respond_to?(:'host') ? example.send(:'host')` here? (lower case h.) My tests broke after this change, where RSpec started trying to use my live production API host instead of the test host. `example.host` returns the correct test value for me, but I don't have a `example.Host` method.